app/controllers/simple.coffee

```diff
@@ -36,3 +36,14 @@ App.EmberTableSimpleController = Ember.Controller.extend
       close: Math.random() * 100 - 50
       volume: Math.random() * 1000000
   .property 'numRows'
+
+  actions:
```

Comment:
I guess you are making it easy for us to test your change; just need to remember to remove these before we merge.